node-src/index.test.ts

```diff
@@ -43,7 +43,7 @@ vi.mock('execa');
 // NOTE: we'd prefer to mock the require.resolve() of `@chromatic-com/playwright/..` but
 // vitest doesn't allow you to do that.
 const mockedBuildCommand = 'mocked build command';
-vi.mock(import('./lib/e2e'), async (importOriginal) => ({
+(vi as any).mock(import('./lib/e2e'), async (importOriginal) => ({
```

Comment:
Is this needed?

Reply:
It might not be after correcting my botched yarn.lock stuff. I'll check!